docs/src/pages/[platform]/connected-components/liveness/customization/Customizationi18n.tsx

```diff
@@ -6,8 +6,8 @@ const dictionary = {
   // use default strings for english
   en: null,
   es: {
-    photosensitivyWarningHeadingText: 'Advertencia de fotosensibilidad',
```

Comment:
Is this a breaking change? Can you confirm it's not written this way in the codebase or that we have a fallback to use the correct spelling version?

Reply:
This is fixed now, right?

Reply:
The fix is merged but not released. This usage is what currently works on latest, and the fix is to make incorrect and correct spellings both work